.github/workflows/spring-backend.yml

```
@@ -0,0 +1,24 @@
+  push:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+        name: Log in to DockerHub  
+        uses: docker/login-action@v2  
```

actionlint (1.7.7)

key "uses" is duplicated in element of "steps" section. previously defined at line:12,col:9

(syntax-check)